### ISSUE TITLE
Mod failing to load with a non-Steam install

### ISSUE THREAD

**Issue by SachoM69** [PippleCultist/HoloCureMultiplayerMod#8]
When running a standalone Holocure install with mod versions 1.1.0 onwards, the mod fails to load. The log does not contain any errors.

Versions up to 1.0.7 start without issues.

**Comment by SachoM69** [comment]
From looking at source code, the problem seems to be in `ModuleInitialize` function:

```
if (!SteamAPI_Init())
{
  g_ModuleInterface->Print(CM_RED, "Couldn't initialize Steam api");
  return AURIE_EXTERNAL_ERROR;
}
```

- The mod prematurely exits if the game is not running from Steam
- the `g_ModuleInterface` is used before it is initialized, so it fails to log the error

**Comment by PippleCultist** [comment]
This is now fixed in v1.1.3
